.github/actions/initialize/action.yml

```diff
@@ -30,5 +30,8 @@ runs:
 
         foreach ($moduleName in (Get-ChildItem -Directory | select Name))
         {
-            Import-Module $moduleName.Name
+            if (Test-Path -Path $moduleName/$moduleName.psm1)
+            {
+                Import-Module $moduleName.Name
+            }
```

Comment:
I forgot to ask under the other PR, but is this explicit import needed for sure?

Reply:
Yes, without it it will throw an error that the module is found but it's not imported use "Import-Module  module name".